README.md

```diff
@@ -16,112 +16,688 @@ If your platform does not support one of the required features, it is also possi
 
```

Comment:
I think it would be good to have the recommended Node version (v18) somewhere in the Local Setup section.

Reply:
Is that normal to have in docs? I usually just look at the package.json. If you can link a good example from another library then I'll copy the same format 🙏